src/App.js

```diff
@@ -1,7 +1,12 @@
-import React from "react";
+import React from 'react';
+import LoadUsers from './Components/UserList';
 
 function App() {
-  return <div>Good luck with the test students!</div>;
+  return (
+    <div>
+      <LoadUsers></LoadUsers>
```

Comment:
should be `<LoadUsers />`.
Also if you `App` components is just implementing another component then rather implment in the `App` component, no need for unnecessary component layer